app/form_objects/support/new_user_school_form.rb

```diff
@@ -0,0 +1,16 @@
+class Support::NewUserSchoolForm
+  include ActiveModel::Model
+
+  attr_accessor :user, :name_or_urn
+
+  def initialize(user:, name_or_urn: nil)
+    @user = user
+    @name_or_urn = name_or_urn
+  end
+
+  def matching_schools
+    School.includes(:responsible_body)
+          .where('urn = ? OR LOWER(name) LIKE(?)', @name_or_urn.to_i, "%#{@name_or_urn.downcase}%")
```

Comment:
would `ILIKE` work here?

Reply:
oooh, good call!
 always try to use standard SQL by default rather than vendor-specific extensions, but I think in this case:

1. we gain clarity
2. it's easy to port it back to a standards-compliant way if needed
3. there'll be a record of this commit in the log if we ever _do_ need to change DBs
4. .....how likely is it that we'll ever actually port this to another DB anyway? (not very)

So yeah, done - thanks!